pages/common/git-subtree.md

```diff
@@ -0,0 +1,24 @@
+# git subtree
+
+> Merge subtrees together and split repository into subtrees.
```

Comment:
```suggestion
> Tool to manage project dependencies as subprojects
```
Hi @einverne what about?

Reply:
Yeah, I agree, the origin description is hard to understand especially for someone use this feature first time.